src/views/kbv/prove-identity-another-way.njk

```diff
@@ -0,0 +1,51 @@
+{% extends "base-form.njk" %}
+{% from "hmpo-radios/macro.njk" import hmpoRadios %}
+{% from "hmpo-warning-text/macro.njk" import hmpoWarningText %}
+
+{% set hmpoPageKey = "prove-identity-another-way" %}
+{% set gtmJourney = "kbv-hmrc - pyi" %}
+{% block mainContentBody %}
+  {{ hmpoHtml(translate("pages.prove-identity-another-way.content")) }}
+
+   {% call hmpoForm(ctx) %}
+    {{ hmpoRadios(ctx, {
+        id: "prove-identity-another-way",
+        name: "prove-identity-another-way",
+        label: translate("pages.prove-identity-another-way.legend"),
+        legend: translate("pages.prove-identity-another-way.legend"),
+        hint: {
+          html: " "
+        },
+        fieldset: {
+          legend: {
+            isPageHeading: true,
+            classes: "govuk-fieldset__legend--m"
+          }
+        },
+        items: [
+          {
+            id: "continue",
+            value: "continue",
+            text: translate("pages.prove-identity-another-way.continue"),
+            hint: {
+              html: " "
+            }
+          },
+          {
+            id : "stop",
+            value: "stop",
+            text: translate("pages.prove-identity-another-way.stop"),
+            hint: {
+              html: " "
+            }
+          }
+        ]
+      }) }}
```

Comment:
Can we implement this using a presenter, like https://github.com/govuk-one-login/ipv-cri-kbv-front/blob/main/src/presenters/question-to-radios.js? Then we can test outside of the template